documentation/decisions/adr/000n-ticket-tracking.md

```diff
@@ -0,0 +1,199 @@
+- **Reporting:** Users can report on key metrics like burndown, point allocation, etc. from directly within the tool
+- **Custom Views:** Users can create custom views for managing tickets with multiple layouts (e.g. kanban board, tabular, roadmap)
+- **Custom Fields:** Users can create custom fields and views to manage their projects
+- **Automation:** Users can automate basic workflows like adding and moving tickets, linking PRs to their originating PRs, etc.
+- **Open Source:** The tool used to manage and host the wiki content should be open source, if possible
+
+## Options Considered
+
+- [GitHub Issues + Zenhub](#github-issues--zenhub)
+- [GitHub Issues + GitHub Projects](#github-issues--github-projects)
+- [Jira](#jira)
+- [OpenProject](#openproject)
+
+## Decision Outcome <!-- REQUIRED -->
+
+We are planning to use **GitHub issues with GitHub projects** because it is the only solution that allows members of the public to submit feature requests and bug reports and then track how those issues are being prioritized within upcoming sprints. Additionally, the use of these tools is free with public repositories and offers other helpful features such as custom fields and views.
+
+**NOTE:** Given some of the constraints around reporting, we may want to continue to explore options that extend the reporting capabilities of GitHub projects. Similarly, if team productivity is meaningfully affected by the absence of the more robust features that Jira or Zenhub offers, we may want to re-evaluate this decision after an initial trial period.
+
+### Positive Consequences <!-- OPTIONAL -->
+
+- We do not need to purchase licenses or seek ATO approval to start begin tracking tickets and planning sprints
+- Members of the public can submit feature requests or bug tickets and track how those requests are being prioritized and worked on
+- We can manage all of our sprint planning and development within the same platform
+- We can extend GitHub project functionality with custom-built automations
+
+### Negative Consequences <!-- OPTIONAL -->
+
+- We will need to develop custom reports to track some of the same metrics that Jira or Confluence offers out of the box
+- We will need to spend a bit more time setting up the GitHub project to replicate some of the features that Jira or Confluence offers (e.g. story points, sprints, epics, etc.)
+- Team members who are familiar with Jira and Zenhub will need to spend a bit more time becoming familiar with GitHub Projects
+
+## Comparison Matrix
+
+- ✅ Feature available, meets requirement
+- ❌ Feature not available, does not meet requirement
+- 🔄 Partial feature, limited feature availability, feature in progress or undergoing improvements
+- 1-3 Strength level
+- ❓Unknown
+
+| Factor                    |    Zenhub     | GitHub Projects |     Jira      |  OpenProject  |
+| ------------------------- | :-----------: | :-------------: | :-----------: | :-----------: |
```

Comment:
Nit: I think there are some IDE tools that will auto-align these markdown tables for you so that the columns are the same width through-out.

Reply:
@lucasmbrown-usds I actually do use a VSCode extension that automatically adjusts table columns ([Markdown table](https://marketplace.visualstudio.com/items?itemName=TakumiI.markdowntable)) the problem with this table is that each of the emojis has a different unicode width which is why the spacing doesn't appear uniform.

Reply:
Oooh I see!